vars/cloudfront.var.yml

```diff
@@ -5,7 +5,7 @@ cloudformation_stacks:
   - name: "{{ service_name }}-{{ Environment }}-CLOUDFRONT"
     cloudformation_waf: "CKAN-{{ Environment }}-WAF"
     state: "{{ state | default('present')}}"
-    region: "{{ region }}"
```

Comment:
be careful with this, are you absolutely certain this was made in us-east-1 as it will have big issues if you try and create cloudfront that was made in sydney. 

You can create cloudfront in any region and it will always go into us-east-1. You can't do this for lambda@edge which 'must' be locked down to us-east-1. 

We have done this in the past by creating the lambda function in a cfn and then referencing it in the cfn template that in sydney,

Looking at the current stacks, this 'will' have issues. since 
https://github.com/qld-gov-au/ckan-qld-infrastructure/blob/master/vars/cloudfront.var.yml#L8 is in sydney, and https://github.com/qld-gov-au/ckan-qld-infrastructure/blob/master/vars/cloudfront-lambda-at-edge.var.yml is in us-east-1 and direct reference. 

Also have you verified that we are not referencing the waf_web_acl in the alb/elb?

Reply:
This has been altered to keep the CloudFormation stack in ap-southeast-2.